nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/api/ESClientInitializationService.java

```diff
@@ -0,0 +1,16 @@
+package org.nuxeo.elasticsearch.api;
+
+import org.elasticsearch.client.transport.TransportClient;
+import org.elasticsearch.common.settings.Settings;
+import org.nuxeo.elasticsearch.config.ElasticSearchRemoteConfig;
+
+/**
+ * @since 9.1
```

Comment:
This is public so please add a short sentence in the Javadoc explaining what this does.